src/components/KTable/KTable.vue

```diff
@@ -125,7 +148,16 @@
                   icon="chevronDown"
                   :size="KUI_ICON_SIZE_20"
                 />
-              </span>
+
+                <div
+                  v-if="resizeColumns && index !== tableHeaders.length - 1"
+                  class="resize-handle"
+                  @click.stop
+                  @mousedown="(evt) => startResize(evt, column.key)"
```

Comment:
(just providing info)

for these native events, passing in with an arrow function works, but you can also use the special `$event` variable if it makes things simpler:


```ts
@mousedown="startResize($event, column.key)"
```

[Docs](https://vuejs.org/guide/essentials/event-handling.html#accessing-event-argument-in-inline-handlers)